Content.Shared/Hands/Components/SharedHandsComponent.cs

```diff
@@ -785,15 +785,25 @@ public bool PutInHand(SharedItemComponent item, bool checkActionBlocker = true)
             return TryPutInActiveHandOrAny(item.Owner, checkActionBlocker);
         }
 
+        /// <summary>
+        ///     Tries to pick up an entity into the active hand. If it cannot, tries to pick up the entity into each other hand.
+        /// </summary>
+        public bool PutInHand(EntityUid uid, bool checkActionBlocker = true)
+        {
+            return TryPutInActiveHandOrAny(uid, checkActionBlocker);
+        }
```

Comment:
So `PutInHand` should either just be removed, or `TryPutInActiveHandOrAny` should just be renamed.

If we want to rename `TryPutInActiveHandOrAny` then this is fine, and we can just rename it after my hands pr is done. Otherwise, this should probably be removed.